### PR TITLE
API-11468 lc sdk go neo ws

### PR DESCRIPTION
**Pull request by maciejjwojcik**

### CODE REVIEW

File: webhooks/testdata/agent_updated.json [hunk 1]
```diff
@@ -6,14 +6,21 @@
...
-				"start": "07:30",
-				"end": "21:30"
-			}
+			"timezone": "Europe/Warsaw",
+			"schedule": [
+				{
+				  "day": "monday",
+				  "enabled": true,
+				  "start": "08:30",
+				  "end": "12:30"
```
Comment (grz-gajda) [inline webhooks/testdata/agent_updated.json]: Formatting.

File: webhooks/testdata/agent_created.json [hunk 2]
```diff
@@ -31,10 +31,15 @@
...
-				"start": "08:30",
-				"end": "12:30"
-			}
+			"timezone": "Europe/Warsaw",
+			"schedule": [
+				{
+				  "day": "monday",
+				  "enabled": true,
+				  "start": "08:30",
+				  "end": "12:30"
```
Comment (grz-gajda) [inline webhooks/testdata/agent_created.json]: Formatting.

File: configuration/structures.go [hunk 3]
```diff
@@ -164,12 +164,17 @@ type AgentFields struct {
...
+	Schedule []Schedule `json:"schedule"`
+}
 
-// WorkSchedulerDay represents single day work schedule
-type WorkSchedulerDay struct {
-	Start string `json:"start"`
-	End   string `json:"end"`
+// Schedule represent a single day work schedule
+type Schedule struct {
+	Enabled bool    `json:"enabled"`
```
Comment (grz-gajda) [inline configuration/structures.go:174]: I think it's optional param but I am not sure. Maybe `*bool`? Look at [that line](https://github.com/livechat/api/blob/70b63fa2f01c85adf4edc783341883e5c16c427b/services/configuration-api/parsers/v3.5/objects.go#L32).

File: webhooks/testdata/agent_updated.json (hunk 1 above)
Comment (maciejjwojcik) [reply webhooks/testdata/agent_updated.json]: Fixed formatting.

File: webhooks/testdata/agent_created.json (hunk 2 above)
Comment (maciejjwojcik) [reply webhooks/testdata/agent_created.json]: Fixed formatting.

File: configuration/structures.go (hunk 3 above)
Comment (maciejjwojcik) [reply configuration/structures.go:174]: Enabled is a required param. Tried making the call without it which resulted in:
```
{
    "error": {
        "type": "validation",
        "message": "`work_scheduler.schedule.enabled` is required"
    }
}
```

Comment (maciejjwojcik) [reply configuration/structures.go:174]: I left `bool` as it was. With `*bool` the tests were failing.  `handler_test.go:166: Payload incorrectly parsed for agent_updated, error: webhook handler error: WorkScheduler.Schedule[0].Enabled mismatch, actual: 0x1400023a4e6, expected: true`

For it to work, I'd need to change [this line](https://github.com/livechat/lc-sdk-go/blob/4fcc7ff3df04f2a8d64b9ec9b26d3ded91046719/webhooks/helpers_test.go#L405) to expect a pointer, which wouldn't make sense, I think?

**Review by grz-gajda — COMMENTED** [review]
Fix unit tests.

**Review by grz-gajda — APPROVED** [review]

**Pull request merged**